src/client/resources/mig.ts

```diff
@@ -21,7 +21,7 @@ export class MigResource extends NamespacedResource {
 
 export enum MigResourceKind {
   MigPlan = 'migplans',
-  MigStorage = 'migstorage',
+  MigStorage = 'migstorages',
```

Comment:
Nice, let's chat about this part of my abstraction, maybe it's a bit weird and could be improved :+1: